src/test/scala/kanaloa/reactive/dispatcher/queue/QueueSpec.scala

```diff
@@ -1,14 +1,18 @@
-package com.iheart.workpipeline.akka.patterns.queue
+package kanaloa.reactive.dispatcher.queue
 
 import akka.actor._
 import akka.testkit.{ TestActorRef, TestProbe }
-import com.iheart.workpipeline.akka.{ SpecWithActorSystem, patterns }
-import com.iheart.workpipeline.akka.patterns.CommonProtocol.{ ShutdownSuccessfully, QueryStatus }
-import com.iheart.workpipeline.metrics.{ Metric, MetricsCollector, NoOpMetricsCollector }
+import kanaloa.reactive.dispatcher
+import kanaloa.reactive.dispatcher.{ SpecWithActorSystem, ApiProtocol }
+import kanaloa.reactive.dispatcher.queue.queue.{ QueueProcessorRef, QueueRef }
+import ApiProtocol.ShutdownSuccessfully
+import kanaloa.reactive.dispatcher.metrics.{ Metric, NoOpMetricsCollector, MetricsCollector }
+import dispatcher.ApiProtocol.{ ShutdownSuccessfully, QueryStatus }
 import Queue._
 import QueueProcessor._
-import com.iheart.workpipeline.akka.patterns.queue.Queue.{ QueueStatus, WorkEnqueued }
-import com.iheart.workpipeline.akka.patterns.queue.QueueProcessor.{ Shutdown }
+import Queue.{ QueueStatus, WorkEnqueued }
+import QueueProcessor.{ Shutdown }
```

Comment:
these should already be handled by the `import Queue._` and `import QueueProcessor._` above? (and the import should probably be the full package name, as mentioned in another comment)

anyway, PR looks ok other than some package naming issues and imports